modules/test/playwright/tests/configuration-admin-web/instanceSettings.spec.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+import {readFile} from 'fs/promises';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {InstanceSettingsPage} from '../../pages/configuration-admin-web/InstanceSettingsPage';
+
+export const test = mergeTests(loginTest());
+
+test('Asserts that a user can export a configuration', async ({page}) => {
+	const instanceSettingsPage = new InstanceSettingsPage(page);
+
+	await instanceSettingsPage.goToInstanceSetting('Accounts', 'Email Domains');
+
+	const emailDomainValidationSwitcher = page.getByRole('switch', {
+		name: 'Enable Email Domain Validation',
+	});
+
+	const saveButton = page
```

Comment:
The save button should be moved to InstanceSettingsPage since it's generic for all instance settings.